backend/app/services/llm/providers/eai.py

```
@@ -0,0 +1,130 @@
+import logging
+import os
+from typing import Any
+
+import io
+import base64
+from elevenlabs.client import ElevenLabs
+from elevenlabs import speech_to_text
+from typing import Any, Dict, Optional, Tuple
```

Ruff (0.15.4)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)